.all-contributorsrc

```diff
@@ -0,0 +1,24 @@
+{
+  "files": [
+    "README.md"
+  ],
+  "imageSize": 100,
+  "commit": false,
+  "contributors": [
+    {
+      "login": "eleith",
+      "name": "eleith",
+      "avatar_url": "https://avatars.githubusercontent.com/u/284832?v=4",
+      "profile": "http://eleith.com",
+      "contributions": [
+        "infra"
+      ]
+    }
+  ],
+  "contributorsPerLine": 7,
+  "projectName": "docker-mailserver",
+  "projectOwner": "docker-mailserver",
+  "repoType": "github",
+  "repoHost": "https://github.com",
+  "skipCi": true
```

Comment:
```suggestion
  "skipCi": false
```

This needs to be false as we have a hard requirement of at least a passing "lint" ci run. But this shouldn't be a problem as we can exclude this file from triggers if some strange behavior occurs.

Reply:
👍🏼